### PR TITLE
Update upcoming section of cboard to scroll vertically

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Quick fix to change Upcoming Section to scroll vertically because the upcoming flyer cells were too small in the original horizontal scroll view

## Changes Made

- Use `FlyerCellPast` to display upcoming flyers in `upcomingSection` of `FlyersView.swift`
- Remove horizontal scroll view from `upcomingSection`

## Test Coverage

Playtested in simulator

## Next Steps

- Update `FlyerCellUpcoming` once we receive finalized designs based on user research and replace `FlyerCellPast` in `upcomingSection`
- Update layout of `upcomingSection` again if necessary

## Screenshots
| Before | After |
| - | - |
|||

### CODE REVIEW

**Comment by vinnie4k** [comment]
Looks good. Ideally, we would create a reusable Section view for upcoming and past since they're the same, but should be good for now. Next time, can you include in the PR other stuff that you tested for such as changing the filters, bookmarking, etc.? Also, are you planning on shipping this right now to the AppStore? I'll approve once you respond.

**Review by tiffany-pann — APPROVED** [review]
LGTM

**Comment by vinnie4k** [comment]
@jjennifergu After hours of figuring out the bash script, I believe I was able to get everything working but I need to have something to test. Because this PR is small enough, I want you to create a new branch off of `release`. You will then add all of these changes into that new branch. This is because resolving merge conflicts will be painful since I configured a lot of the dependency stuff.

You will also need to change the version number to `1.3.7` and build number to `25`. You can do this by clicking on "Volume" in the Project Navigator, "Volume" under Targets, and then the General tab. Once you are done, recreate the PR **to merge it to `release`** and add me as a reviewer. I configured it so that it would automatically submit it to TestFlight instead of the AppStore since we are still testing. If everything works properly, close this PR and delete the branch.

In terms of how this repository will work, if you ever want to make a change on the released app, create a branch off of `release` and make a PR to merge it to `release` (make sure to increment the build number and version number). All development work will be on the `master` branch. Once we are done with the dev side, we can merge `master` to `release`.  I'm not exactly sure if this will work but I configured the bash script and workflow to be able to convert the dev stuff to prod.

@tiffany-pann When you get the chance, can you prevent the `release` branch from having direct commits (i.e. require a PR in order to change it)?

**Pull request closed without merging**
